sdv/single_table/base.py

```diff
@@ -112,7 +112,9 @@ def __init__(
     ):
         self._validate_inputs(enforce_min_max_values, enforce_rounding)
         self.metadata = metadata
+        self.table_name = Metadata.DEFAULT_SINGLE_TABLE_NAME
```

Comment:
can we make this a private attribute?
```suggestion
        self._table_name = Metadata.DEFAULT_SINGLE_TABLE_NAME
```